src/XamlX/Transform/Transformers/XArgumentsTransformer.cs

```diff
@@ -15,9 +15,10 @@ public IXamlAstNode Transform(AstTransformationContext context, IXamlAstNode nod
                 var argDirectives = ni.Children.OfType<XamlAstObjectNode>()
                     .Where(d => d.Type is XamlAstXmlTypeReference xref
                                 && xref.XmlNamespace == XamlNamespaces.Xaml2006 && xref.Name == "Arguments").ToList();
-                if (argDirectives.Count > 1 && context.StrictMode)
```

Comment:
The exception has served as an "early escape" here. We need to be sure that node processing is properly halted after error is reported.

Reply:
Fixed